### PR TITLE
Add shortcut in navigation bar for featured projects

### PR DESCRIPTION
**Pull request by anjali-sharma**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -25,9 +25,9 @@ def draw(routes_name)
   
   resource :user_sessions, only: [:create]
   resources :newsletters, only: [:create]
-  get  "first_edition" => "home#first_edition_projects"
-  get  "second_edition" => "home#second_edition_projects"
-  get  "third_edition" => "home#third_edition_projects"
+  get  "featured_first_edition_projects" => "home#first_edition_projects"
```
Comment (DMips) [inline config/routes.rb]: woah that's long name, how about `first_edition_projects`? also did you make sure we're not using eg `first_edition_path` anywhere else? I'm fairly sure we're using it in home view (index.html)

File: app/views/layouts/_header.html.slim [hunk 2]
```diff
@@ -52,6 +52,15 @@
                 = link_to "Swift", swift_learning_materials_path
               li
                 = link_to "VB.NET", vb_dot_net_learning_materials_path
+          li.dropdown.no-image
```
Comment (DMips) [inline app/views/layouts/_header.html.slim]: does it look ok on the desktop? Space wise that is. We have already quite a few items in the header

File: config/routes.rb (hunk 1 above)
Comment (anjali-sharma) [reply config/routes.rb]: oh yeah... thanks for pointing out.

File: app/views/layouts/_header.html.slim (hunk 2 above)
Comment (anjali-sharma) [reply app/views/layouts/_header.html.slim]: looks fine on mine. But yeah things can be better categorized and refactored. Let me try and do that.

**Pull request closed without merging**